plugins/jobs/src/js/components/form/ArgsSection.tsx

```diff
@@ -35,14 +37,29 @@ class ArgsSection extends React.Component<ArgsSectionProps> {
       showErrors
     } = this.props;
 
+    const argTooltipContent = (
+      <Trans>
+        Arguments are a Docker only feature and will pass docker arguments to
+        the container at startup.
+      </Trans>
+    );
+
     return args.map((arg: string, index: number) => {
       let label = null;
       if (index === 0) {
         label = (
           <FieldLabel>
             <FormGroupHeading>
               <FormGroupHeadingContent primary={true}>
-                <Trans>Arg</Trans>
+                <Trans>Arguments</Trans>{" "}
+                <Tooltip
+                  content={argTooltipContent}
+                  interactive={true}
```

Comment:
I was trying to figure out what this prop does. It's not documented so I read the source code and even tried removing it but nothing changed. Maybe you can explain it to me :D

Reply:
hmmm... 🤷‍♂ 

copy 🍝  from [here](https://github.com/dcos/dcos-ui/blob/master/plugins/jobs/src/js/components/form/GeneralFormSection.tsx#L69-L76)